packages/components/PrintStatementPanel/PrintStatementPanel.module.css

```diff
@@ -0,0 +1,118 @@
+.MessagePrefixBadgeUnicorn,
+.MessagePrefixBadgeSolidColor {
+  flex-shrink: 0;
+  align-self: center;
+}
+.MessagePrefixBadgeUnicorn {
+  width: 1.25rem;
+  height: 1.25rem;
+
+  /* HACK to horizontally align text; this icon needs to match the MessageIcon width. */
+  margin: 0 calc((18px - 1.25rem) / 2);
+}
+.MessagePrefixBadgeSolidColor {
+  border-radius: 100%;
+  width: 0.5rem;
+  height: 0.5rem;
+
+  /* HACK to horizontally align text; this icon needs to match the MessageIcon width. */
```

Comment:
Seems like we could probably set up some variables here or something? That seems both easier to maintain and more explanatory than a comment.

Reply:
Oh yeah, this is ripe for cleanup 😄 just initial copy/paste, but yeah, agree! We can simplify this 😎